installation/scripts/installer.sh

```diff
@@ -96,7 +96,7 @@ kubectl apply -f - <<<"$COMBO_YAML"
 if [ $KNATIVE ]; then
     kubectl -n kyma-installer patch configmap installation-config-overrides -p '{"data": {"global.knative": "true"}}'
     kubectl -n kyma-installer patch configmap installation-config-overrides -p '{"data": {"global.kymaEventBus": "false"}}'
-    kubectl -n kyma-installer patch configmap nats-streaming-overrides -p '{"data": {"global.natsStreaming.clusterID": "kyma-nats-streaming"}}'
+    kubectl -n kyma-installer patch configmap installation-config-overrides -p '{"data": {"global.natsStreaming.clusterID": "knative-nats-streaming"}}'
```

Comment:
Why do you need this override?

Reply:
Because, in the case of knative, we need to change the NATS Streaming Cluster id *not* to have the default `kyma-` prefix but to have the `knative-` prefix instead.